doc/nrf/dm_managing_code.rst

```diff
@@ -77,10 +77,10 @@ Refer to the publicly available `Git`_ documentation.
 Forking a repository of the |NCS|
 *********************************
 
-In some cases, you might want to keep a :ref:`soft fork <dm-glossary>` of one or more repositories that are part of the |NCS|.
+In some cases, you might want to keep a :ref:`soft fork <glossary>` of one or more repositories that are part of the |NCS|.
```

Comment:
@ru-fu why aren't we using the built-in `:term:` role? https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-term

Reply:
@mbolivar-nordic to use :term: the glossary page has to use the glossary:: directive. I'll implement it here and see if it works correctly.

Reply:
It works. Fixed.